mne/epochs.py

```diff
@@ -1599,6 +1600,29 @@ def __repr__(self):
         class_name = 'Epochs' if class_name == 'BaseEpochs' else class_name
         return '<%s | %s>' % (class_name, s)
 
+    def _repr_html_(self):
+        if self.baseline is None:
+            baseline = 'off'
+        else:
+            baseline = tuple(['None' if b is None else ('%.3f' % b)
+                              for b in self.baseline])
```

Comment:
Actually, we shouldn't have ``None`` values in `Epochs.baseline` anymore these days, as we're replacing them with the actual times. Did you come across a situation where this was not the case? Because then it's a bug we need to fix.

In any case: you can (should) drop special-casing `None` here

Reply:
Alright, I will update this part! I have no examples where the baseline is `None`.